app/assets/javascripts/faceted_search_form.js.coffee

```diff
@@ -63,4 +70,4 @@ $('.sort-order ol.dropdown-menu a').click (event)->
 
   if sort_by_field.val() != value
     sort_by_field.val(value)
-    $('form#facets-form').submit()
+    $('form#facets-form').submit()
```

Comment:
oh, you should fix your editor to always put in trailing line endings, and ESPECIALLY not to take them out. A text file without a line ending is technically not a text file according to the POSIX standard. [Check this out](http://stackoverflow.com/a/729795)